### ISSUE TITLE
Tensorboard can be started multiple times for one project

### ISSUE THREAD

**Issue by jrozansk** [polyaxon/polyaxon#51]
**Steps to reproduce:**
1. Create a project.
2. Start tensorboard with polyaxon CLI.
polyaxon project start_tensorboard
3. Use polyaxon CLI again to start another tensorboard within the same project.
4. Delete tensorboard twice.
polyaxon project stop_tensorboard

**Expected results:**
It is up to authors how can it be handled.
Either not allowing creation of second tensorboard or possibility to choose which tensorboard to stop would be ok.

**Actual results:**
Only one tensorboard is stopped. Second remains running. Data scientists would be forced to approach k8s cluster and delete tensorboard manually.

Note: This occurs for version 0.0.3 released as chart.

**Comment by mouradmourafiq** [comment]
Hi @jrozansk, I noticed that issue as well,  also there's security problem involved by giving access to tensorboard over LoadBalancer/Ingress.

In the coming version, I fixed the issue with the security, i.e. tensorboard will be subject to the same permissions sat on the project by the user, and will not be visible on a separate LoadBlancer/Ingress.

I also added the possibility to start jupyter notebooks in the same fashion.

More or less the logic for creating tensorboards / jupyter notebooks is different, but I will keep in mind to test this use case again.
